test/integration/copy_paste_spec.js

```diff
@@ -117,4 +117,47 @@ describe("Copy and paste", () => {
       );
     });
   });
+  describe("all text", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait("copy_paste_ligatures.pdf", ".textLayer");
+      await mockClipboard(pages);
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must check that we've all the contents on copy/paste", async () => {
+      await Promise.all(
+        pages.map(async ([browserName, page]) => {
+          await page.keyboard.down("Control");
+          await page.keyboard.press("a");
+          await page.keyboard.up("Control");
+
+          await page.waitForTimeout(100);
+
+          await page.keyboard.down("Control");
+          await page.keyboard.press("c");
+          await page.keyboard.up("Control");
+
+          await page.waitForTimeout(100);
+
+          await page.waitForFunction(
+            `document.querySelector('#viewerContainer').style.cursor !== "wait"`
+          );
+
+          const text = await page.evaluate(() =>
+            navigator.clipboard.readText()
+          );
+
+          expect(!!text).withContext(`In ${browserName}`).toEqual(true);
+          expect(text)
+            .withContext(`In ${browserName}`)
+            .toEqual("abcdeffffiflffifflﬅstghijklmno");
```

Comment:
It looks like there's one non-normalized ligature, ie. `ﬅ` one, left here?

Reply:
In the `master` branch this copies as `"abcdeffffiflffifflſtstghijklmno"`.

Reply:
Yes I know: it works like that in Acrobat.
In Chrome and Foxit, this char is mapped on `ſt` (the first char is a `\u017F`, e.g. long 's') and NFKC changes it to `st`.
We can add it in the list of the char to normalize and then we'll get a `st`.
I'm not an expert in latin, hence I don't know what a user could expect to have.

Reply:
It's annoying that there's no consensus between PDF Viewers here :-(

Given that we previously normalized this and that Chrome also does so (since nowadays it feels like users often compare with Chrome), I'm leaning towards suggesting that we should keep normalizing that ligature.